torchci/pages/kpis.tsx

```diff
@@ -25,7 +26,83 @@ export default function Kpis() {
     
     return (
         <Grid container spacing={2}>
-            <Grid item xs={true} height={ROW_HEIGHT}>
+            <Grid item xs={6} height={ROW_HEIGHT}>
```

Comment:
I feel like it's better to have each of these in it's own row. That way you can correlate different KPIs together by just moving your eyes down vertically. For example, % jobs red and % red commits are correlated so if you want to see them in Feburary, you can just look up and down on the graphs and figure that out quickly.

Reply:
If you want to group them, you can maybe add like a header section in between.

Reply:
I was thinking in terms of optimizing the view on a desktop, so that you can see everything on a single screen.

I do like the idea of correlated charts being stacked vertically though, so I might shuffle these around a bit